grammar.dd

```diff
@@ -832,6 +832,13 @@ $(GNAME AsmInstruction):
     $(D df) $(I Operands)
     $(D dd) $(I Operands)
     $(D de) $(I Operands)
+    $(D db) $(GLINK_LEX StringLiteral)
+    $(D ds) $(GLINK_LEX StringLiteral)
+    $(D di) $(GLINK_LEX StringLiteral)
+    $(D dl) $(GLINK_LEX StringLiteral)
+    $(D df) $(GLINK_LEX StringLiteral)
```

Comment:
Each can have multiple operands. If an operand is a string literal, it is as if there were length operands, where length is the number of characters in the string

Though you're right that this doesn't make sense, and now that I'm reading the parser source it seems that I need to rewrite this part of the spec and also change the grammar.